enhancements/audit-command.md

```diff
@@ -0,0 +1,594 @@
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `SDK` |
+| Language | `helm` |
+| Project Layout | `helm.sdk.operatorframework.io/vX` |
+| SDK Version | Check in the Dockerfile. (e.g see [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/helm/memcached-operator/Dockerfile#L1) where it will be `v1.4.2`) |
+
+- If the attribute `layout of the`PROJECT` file be `go.kubebuilder.io/vX` (see e.g [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/go/v3/memcached-operator/PROJECT#L2)):
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | Check if it has plugins with `sdk.operatorframework.io/` (e.g see [here](https://github.com/operator-framework/operator-sdk/blob/v1.4.2/testdata/go/v3/memcached-operator/PROJECT#L12-L14)) to set as `SDK` otherwise it will be `Kubebuilder`|
+| Language | `golang` |
+| Project Layout | `go.kubebuilder.io/vX` |
+
+However, note that for the previous versions of projects built with `Kubebuilder` we will not found the layout information in the `PROJECT` file as well. (e.g see [here](https://github.com/kubernetes-sigs/kubebuilder/blob/v2.3.2/testdata/project-v2/PROJECT)) then:
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `Kubebuilder`|
+| Language | `golang` |
+| Project Layout | If the attribute `version` in the `PROJECT` file be `2` it will be `go.kubebuilder.io/v2` otherwise, it will be `go.kubebuilder.io/v1` |
+
+**Gathering the info from go.mod file**
+
+Now, if the repository has not a `PROJECT` file then, it means that the project could also be built with an SDK version < 1.0 or any other tool which is not SDK or Kubebuilder. In this way, check if the project has a `go.mod` file with the sdk dependency `github.com/operator-framework/operator-sdk`: 
+
+```go
+go 1.13
+
+require (
+	github.com/operator-framework/operator-sdk v0.17.0
+```
+
+Following how we would set the values for the above example. 
+
+| Column | Description | 
+| ------ | ----- | 
+| Builder | `SDK`|
+| Language | `golang` |
+| SDK Version | `v0.17.0` |
+| Golang Version | `1.13` |
+
+**NOTE** If the `go.mod` was found but not the SDK dependency we will set the `Language` as `golang`. Also, we will always check the `go.mod` file for Golang projects to know the Golang version used.  
+
+**Gathering the info from build/Dockerfile**
+
+You will only need to check the `build/Dockerfile` when the project is not writen in Go. In this way, when the `go.mod` file cannot to be found then, it might to be an operator project using Helm or Ansible which was built with an SDK version < 1.0. In this case, we will be looking for `quay.io/operator-framework/ansible-operator` or `quay.io/operator-framework/helm-operator` images. Let's check an Ansible Operator. See the [Dockerfile](https://github.com/operator-framework/operator-sdk-samples/blob/v0.17.0/ansible/memcached-operator/build/Dockerfile): 
```

Comment:
Similarly infeasible as `PROJECT`.

Reply:
See that the Dockerfile is only a text file that is parsable. Also, as described above we can check a similar implementation working successfully and doing that. See: https://github.com/operator-framework/enhancements/pull/66#discussion_r585680068

So, I am closing this one.

Reply:
Ditto: https://github.com/operator-framework/enhancements/pull/66#discussion_r586672948